frontend/packages/integration-tests-cypress/tests/crud/k8-openshift-cruds.spec.ts

```diff
@@ -97,6 +97,9 @@ describe('Kubernetes resource CRUD operations', () => {
 
   testObjs.forEach(
     ({ kind, namespaced = true, humanizeKind = true, testI18n = true }, resource) => {
+      if (Cypress.env('k8TestResource') && kind !== Cypress.env('k8TestResource')) {
```

Comment:
What is this env var for?

Reply:
debugging, so only one resource is run from the 28.